doc/documentation_data_format.md

```diff
@@ -365,43 +361,25 @@ Additional columns may be added.
     - parameterScaleNormal: mean; standard deviation (**not** variance)
     - parameterScaleLaplace: location; scale
 
+- `objectivePriorType` [STRING, optional]
 
-### Additional optional columns
-
-Extra columns:
-
-- `hierarchicalOptimization` (optional)
-
-  hierarchicalOptimization: 1 if parameter is optimized using hierarchical
-  optimization approach, 0 otherwise.
-
-- `initializationPriorType` (optional)
+  Prior types used for the objective function during optimization or sampling.
+  For possible values, see `initializationPriorType`.
 
-  Prior types used for sampling of initial points for optimization. Uses the
-  entries from `priorType` as default, but will overwrite those, if
-  something else is specified here. For more detailed documentation, see
-  `priorType`.
+- `objectivePriorParameters` [STRING, optional]
 
-- `initializationPriorParameters` (optional)
+  Prior parameters used for the objective function during optimization.
+  For more detailed documentation, see `initializationPriorParameters`.   
 
-  Prior parameters used for sampling of initial points for optimization. Uses
-  the entries from `priorParameters` as default, but will overwrite those, if
-  something else is specified here. For more detailed documentation, see
-  `priorParameters`.
 
-- `objectivePriorType` (optional)
+### Additional optional columns
```

Comment:
Not related to sampling, but I think we wanted to kick that out here, as that will be quite tool-specific.